lib/install/validate/config_to_data.go

```diff
@@ -107,7 +107,7 @@ func setApplianceResources(op trace.Operation, vm *vm.VirtualMachine, d *data.Da
 // setVCHResources will populate the configuration data based on the deployed VCH config
 func setVCHResources(op trace.Operation, vch *object.ResourcePool, d *data.Data) error {
 	var p mo.ResourcePool
-	ps := []string{"config.cpuAllocation", "config.memoryAllocation"}
+	ps := []string{"config"}
```

Comment:
What's the purpose of this change?

Reply:
Achieves the same result w/less characters -- which isn't a great reason...happy to pull it out..

FYI: plan on opening a pr against `govmomi` implementing this functionality via `object.ResourcePool`...(and a few others) then a pr against vic-machine to utilize....

Reply:
I think there are a lot of `config.*` properties other than `config.cpuAllocation` and `config.memoryAllocation`, aren't there?

Reply:
Not really - in addition to those two there are:
- changeVersion: string 
- entity: moref
- lastModified: dateTime

Reply:
What about `config.hardware.numCPU` and `config.hardware.memoryMB` (used in the function above)?

Reply:
I actually thought I switched this back...but regardless this is returning the [resourceConfigSpec](https://vdc-download.vmware.com/vmwb-repository/dcr-public/6b586ed2-655c-49d9-9029-bc416323cb22/fa0b429a-a695-4c11-b7d2-2cbc284049dc/doc/index.html#linkf89ddfb4da9f5fead1008dc6689b547fb425dc2b;vim.ResourcePool.html) which contains the two data objects we were returning and would now include a string, moref & datetime...